### DISCUSSION TITLE
Can Awareness be disabled for a document?

### DISCUSSION THREAD

**Discussion by canadaduane** [ueberdosis/hocuspocus#673] [Q&A] [answered]
Is it possible to disable awareness messages for just a set of documents?

Our use case is that we have a Yjs document per user (to store metadata about that user), and each user can "connect" to many other users' documents. So we have (say) several hundred Yjs documents connected at any given moment, but we don't want all of the Awareness messaging for these documents affecting bandwidth or server-side processes.

**Comment by canadaduane** [comment, marked as answer]
I believe this recent comment from @janthurau answers my question:

https://github.com/ueberdosis/hocuspocus/commit/9cb0b54b90aeb3564f45268cfaeff8d9dc6e3b23

In short, pass "null" as the `awareness` object to HocuspocusProvider and the document will not send awareness info. At least one document per websocket should have awareness enabled, however.

**Reply by janthurau** [reply]
yes! Shortly after you opened this discussion here, we got a PR from @raineorshine :-) (https://github.com/ueberdosis/hocuspocus/pull/675)
